app/views/unattended/provisioning_templates/iPXE/ipxe_default_local_boot.erb

```diff
@@ -3,6 +3,8 @@ kind: iPXE
 model: ProvisioningTemplate
 name: iPXE default local boot
 snippet: false
+description: |
+  TODO
```

Comment:
@lzap could you please provide a suggestion?

Reply:
Just copy PXELinux local boot description and replace with iPXE. The very same purpose.